### PR TITLE
feat(port): support Sway

### PR DESCRIPTION
**Pull request by Hendrikto**
#### Description

This PR adds a port to the Sway Wayland compositor.

#### Assets

1. An image to the icon of the app that this port is related to:
   Sway logo
3. A screenshot with a good zoom in fullscreen showing this port in action:
   sway port fullscreen

#### Checklist

- [X] PR description included
- [X] I've read the [documents](https://github.com/daltonmenezes/aura-theme#documentation)
- [X] I've created or commented in an issue related to this port asking to work on it
- [X] I know I shouldn't change any files in the packages folder manually
- [X] I've added this port at the end of the related category list in the [main README](https://github.com/daltonmenezes/aura-theme/blob/main/README.md)
- [X] I've attached an image to the icon of the app that this port is related to
- [X] I've attached a screenshot with a good zoom in fullscreen showing this port in action

### CODE REVIEW

**Comment by daltonmenezes** [comment]
@Hendrikto 

icon

banner

**Comment by Hendrikto** [comment]
Hey, I got sidetracked for a bit. Thanks for your patience. I think the PR should be ready to merge now.

File: src/ports/sway/templates/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      </p></td>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/Hendrikto">Hendrikto</a></td>
+    </tr>
+  </tbody>
+</table>
```
Comment (daltonmenezes) [inline src/ports/sway/templates/README.md:30]: Please, don't remove the **author-thead** and **author-tbody**, keep it as shown in [Publishing Ports doc](https://github.com/Hendrikto/aura-theme/blob/main/docs/PUBLISHING_PORTS.md):
```html
<table>
  <thead>
    <tr>
      <td valign="bottom"><p align="center">
        <a href="https://github.com/your-username">
          <img src="https://github.com/your-username.png?size=100" align="center" />
        </a>
      </p></td>
      {{{ author-thead }}}
    </tr>
  </thead>

  <tbody>
    <tr>
      <td><a href="https://github.com/your-username">Your name</a></td>
      {{{ author-tbody }}}
    </tr>
  </tbody>
</table>
```

**Review by daltonmenezes — CHANGES_REQUESTED** [review]

File: src/ports/sway/templates/README.md (hunk 1 above)
Comment (Hendrikto) [reply src/ports/sway/templates/README.md]: Fixed.

**Comment by daltonmenezes** [comment]
Thanks @Hendrikto 💜

**Pull request merged**